xbmc/pvr/windows/GUIWindowPVRRecordings.cpp

```diff
@@ -262,11 +262,34 @@ bool CGUIWindowPVRRecordings::OnMessage(CGUIMessage &message)
           {
             case ACTION_SELECT_ITEM:
             case ACTION_MOUSE_LEFT_CLICK:
+              switch(CSettings::GetInstance().GetInt(CSettings::SETTING_MYVIDEOS_SELECTACTION))
```

Comment:
I know but I think this does not make too much sense, because we then would do the call for every ACTION_*, even for those we don't handle.